cpp/oneapi/dal/algo/jaccard/backend/cpu/vertex_similarity_default_kernel.cpp

```diff
@@ -540,13 +577,13 @@ similarity_result call_jaccard_block_kernel(const descriptor_base &desc,
         array<std::pair<std::uint32_t, std::uint32_t>>::empty(number_elements_in_block);
```

Comment:
number_elements... - use term count and make its name shorter

Reply:
Will be changed by Alexey with the implementation